tools/cli/tests/start-demo.test.js

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const execSync = require('child_process').execSync;
+const { exec } = require('child_process');
+const path = require('path');
+const rimraf = require('rimraf');
+
+const componentFolder = 'create-component-test-folder';
+const componentName = 'test-name';
+const componentFolderPath = path.resolve(path.join(process.cwd(), componentFolder));

```

Comment:
Maybe use `__dirname` instead because the script can be run from different directories.

Reply:
Changed.